internal/printer/print-to-js.go

```diff
@@ -87,13 +88,18 @@ func printToJs(p *printer, n *Node, cssLen int, opts transform.TransformOptions)
 const whitespace = " \t\r\n\f"
 
 // Returns true if the expression only contains a comment block (e.g. {/* a comment */})
-func expressionOnlyHasCommentBlock(n *Node) bool {
-	clean, _ := removeComments(n.FirstChild.Data)
-	return n.FirstChild.NextSibling == nil &&
+func expressionOnlyHasComment(n *Node) bool {
+	if n.FirstChild == nil {
+		return false
+	}
+	clean, _ := helpers.RemoveComments(n.FirstChild.Data)
```

Comment:
RemoveComments can return an error. Shouldn't we be handling it?

Reply:
Wasn't handled before, but yes we probably should

Reply:
Looking at it again now, I think it's safe to ignore comments. We already handle unterminated js and html comments in the tokenizer.
https://github.com/withastro/compiler/blob/a8a845f84b08dd4a4e50f5dd433f0350155c7937/internal/token.go#L1030-L1039
https://github.com/withastro/compiler/blob/a8a845f84b08dd4a4e50f5dd433f0350155c7937/internal/token.go#L907-L916

I'll refactor the RemoveComments function to only return strings.